### ISSUE TITLE
Syslog messages not being received when using provided Docker Compose file

### ISSUE THREAD

**Issue by melbruki** [librenms/docker#396]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/librenms/docker/blob/master/.github/SUPPORT.md)

### I've found a bug and checked that ...

- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

I set up a syslog server using the Docker Compose file provided in the project's GitHub repository. I followed the instructions precisely, but the server isn't receiving any syslog messages. To test the setup, I even tried sending a syslog message from the Docker host, but nothing comes through.

### Expected behaviour

The test syslog message should be received and logged by the server.

### Actual behaviour

No syslog messages are appearing in librenms syslog tab, including the test message sent from the Docker host.

### Steps to reproduce

Steps to reproduce:

Clone the repository and navigate to the appropriate directory.
Use the provided Docker Compose file to set up the syslog server.
Verify the server is listening on 127.0.0.1:514 using netstat or similar tool.
Send a test syslog message from the Docker host: echo "<14>1 date +'%Y-%m-%dT%H:%M:%S' hostname TEST - - - This is a test syslog message" | nc -u -w1 127.0.0.1 514.
Check the syslog messages.
Expected behavior:
The test syslog message should be received and logged by the server.

### Docker info

```text
Client: Docker Engine - Community
 Version:    24.0.6
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.11.2
    Path:     /usr/libexec/docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.21.0
    Path:     /usr/libexec/docker/cli-plugins/docker-compose

Server:
 Containers: 13
  Running: 8
  Paused: 0
  Stopped: 5
 Images: 6
 Server Version: 24.0.6
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: systemd
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 8165feabfdfe38c65b599c4993d227328c231fca
 runc version: v1.1.8-0-g82f18fe
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: builtin
  cgroupns
 Kernel Version: 6.2.0-1011-aws
 Operating System: Ubuntu 22.04.1 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 2
 Total Memory: 1.876GiB
 Name: ip-172-26-1-22
 ID: 2d251c76-8ff3-47d5-8bed-2d04c7f2be6c
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
```

### Docker Compose config

```yaml
name: librenms
services:
  db:
    command:
    - mysqld
    - --innodb-file-per-table=1
    - --lower-case-table-names=0
    - --character-set-server=utf8mb4
    - --collation-server=utf8mb4_unicode_ci
    container_name: librenms_db
    environment:
      MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
      MYSQL_DATABASE: librenms
      MYSQL_PASSWORD: "97176124"
      MYSQL_USER: librenms
      TZ: UTC
    image: mariadb:10.5
    networks:
      default: null
    restart: always
    volumes:
    - type: bind
      source: /var/librenms/db
      target: /var/lib/mysql
      bind:
        create_host_path: true
  dispatcher:
    cap_add:
    - NET_ADMIN
    - NET_RAW
    container_name: librenms_dispatcher
    depends_on:
      librenms:
        condition: service_started
        required: true
      redis:
        condition: service_started
        required: true
    environment:
      CACHE_DRIVER: redis
      DB_HOST: db
      DB_NAME: librenms
      DB_PASSWORD: "97176124"
      DB_TIMEOUT: "60"
      DB_USER: librenms
      DISPATCHER_NODE_ID: dispatcher1
      LIBRENMS_SNMP_COMMUNITY: librenmsdocker
      LIBRENMS_WEATHERMAP: "false"
      LIBRENMS_WEATHERMAP_SCHEDULE: '*/5 * * * *'
      LOG_IP_VAR: remote_addr
      MAX_INPUT_VARS: "1000"
      MEMORY_LIMIT: 256M
      OPCACHE_MEM_SIZE: "128"
      PGID: "1000"
      PUID: "1000"
      REAL_IP_FROM: 0.0.0.0/32
      REAL_IP_HEADER: X-Forwarded-For
      REDIS_HOST: redis
      SESSION_DRIVER: redis
      SIDECAR_DISPATCHER: "1"
      TZ: UTC
      UPLOAD_MAX_SIZE: 16M
    hostname: librenms-dispatcher
    image: librenms/librenms:latest
    networks:
      default: null
    restart: always
    volumes:
    - type: bind
      source: /var/librenms/librenms
      target: /data
      bind:
        create_host_path: true
  librenms:
    cap_add:
    - NET_ADMIN
    - NET_RAW
    container_name: librenms
    depends_on:
      db:
        condition: service_started
        required: true
      msmtpd:
        condition: service_started
        required: true
      redis:
        condition: service_started
        required: true
    environment:
      CACHE_DRIVER: redis
      DB_HOST: db
      DB_NAME: librenms
      DB_PASSWORD: "97176124"
      DB_TIMEOUT: "60"
      DB_USER: librenms
      LIBRENMS_SNMP_COMMUNITY: librenmsdocker
      LIBRENMS_WEATHERMAP: "false"
      LIBRENMS_WEATHERMAP_SCHEDULE: '*/5 * * * *'
      LOG_IP_VAR: remote_addr
      MAX_INPUT_VARS: "1000"
      MEMORY_LIMIT: 256M
      OPCACHE_MEM_SIZE: "128"
      PGID: "1000"
      PUID: "1000"
      REAL_IP_FROM: 0.0.0.0/32
      REAL_IP_HEADER: X-Forwarded-For
      REDIS_HOST: redis
      SESSION_DRIVER: redis
      TZ: UTC
      UPLOAD_MAX_SIZE: 16M
    hostname: librenms
    image: librenms/librenms:latest
    networks:
      default: null
    ports:
    - target: 8000
      published: "8000"
      protocol: tcp
    restart: always
    volumes:
    - type: bind
      source: /var/librenms/librenms
      target: /data
      bind:
        create_host_path: true
  msmtpd:
    container_name: librenms_msmtpd
    environment:
      SMTP_AUTH: "on"
      SMTP_FROM: foo@gmail.com
      SMTP_HOST: smtp.gmail.com
      SMTP_PASSWORD: bar
      SMTP_PORT: "587"
      SMTP_STARTTLS: "on"
      SMTP_TLS: "on"
      SMTP_TLS_CHECKCERT: "on"
      SMTP_USER: foo
    image: crazymax/msmtpd:latest
    networks:
      default: null
    restart: always
  redis:
    container_name: librenms_redis
    environment:
      TZ: UTC
    image: redis:5.0-alpine
    networks:
      default: null
    restart: always
  snmptrapd:
    cap_add:
    - NET_ADMIN
    - NET_RAW
    container_name: librenms_snmptrapd
    depends_on:
      librenms:
        condition: service_started
        required: true
      redis:
        condition: service_started
        required: true
    environment:
      CACHE_DRIVER: redis
      DB_HOST: db
      DB_NAME: librenms
      DB_PASSWORD: "97176124"
      DB_TIMEOUT: "60"
      DB_USER: librenms
      LIBRENMS_SNMP_COMMUNITY: librenmsdocker
      LIBRENMS_WEATHERMAP: "false"
      LIBRENMS_WEATHERMAP_SCHEDULE: '*/5 * * * *'
      LOG_IP_VAR: remote_addr
      MAX_INPUT_VARS: "1000"
      MEMORY_LIMIT: 256M
      OPCACHE_MEM_SIZE: "128"
      PGID: "1000"
      PUID: "1000"
      REAL_IP_FROM: 0.0.0.0/32
      REAL_IP_HEADER: X-Forwarded-For
      REDIS_HOST: redis
      SESSION_DRIVER: redis
      SIDECAR_SNMPTRAPD: "1"
      TZ: UTC
      UPLOAD_MAX_SIZE: 16M
    hostname: librenms-snmptrapd
    image: librenms/librenms:latest
    networks:
      default: null
    ports:
    - target: 162
      published: "162"
      protocol: tcp
    - target: 162
      published: "162"
      protocol: udp
    restart: always
    volumes:
    - type: bind
      source: /var/librenms/librenms
      target: /data
      bind:
        create_host_path: true
  syslogng:
    cap_add:
    - NET_ADMIN
    - NET_RAW
    container_name: librenms_syslogng
    depends_on:
      librenms:
        condition: service_started
        required: true
      redis:
        condition: service_started
        required: true
    environment:
      CACHE_DRIVER: redis
      DB_HOST: db
      DB_NAME: librenms
      DB_PASSWORD: "97176124"
      DB_TIMEOUT: "60"
      DB_USER: librenms
      LIBRENMS_SNMP_COMMUNITY: librenmsdocker
      LIBRENMS_WEATHERMAP: "false"
      LIBRENMS_WEATHERMAP_SCHEDULE: '*/5 * * * *'
      LOG_IP_VAR: remote_addr
      MAX_INPUT_VARS: "1000"
      MEMORY_LIMIT: 256M
      OPCACHE_MEM_SIZE: "128"
      PGID: "1000"
      PUID: "1000"
      REAL_IP_FROM: 0.0.0.0/32
      REAL_IP_HEADER: X-Forwarded-For
      REDIS_HOST: redis
      SESSION_DRIVER: redis
      SIDECAR_SYSLOGNG: "1"
      TZ: UTC
      UPLOAD_MAX_SIZE: 16M
    hostname: librenms-syslogng
    image: librenms/librenms:latest
    networks:
      default: null
    ports:
    - target: 514
      published: "514"
      protocol: tcp
    - target: 514
      published: "514"
      protocol: udp
    restart: always
    volumes:
    - type: bind
      source: /var/librenms/librenms
      target: /data
      bind:
        create_host_path: true
networks:
  default:
    name: librenms_default
```

### Logs

```text
Command 'docke' not found, did you mean:
  command 'docker' from deb podman-docker (3.4.4+ds1-1ubuntu1.22.04.2)
  command 'docker' from deb docker.io (24.0.5-0ubuntu1~22.04.1)
Try: apt install <deb name>
root@ip-172-26-1-22:/var/librenms# docker ps
CONTAINER ID   IMAGE                              COMMAND                  CREATED          STATUS                    PORTS                                                                                                      NAMES
b1f9f898b91a   librenms/librenms:latest           "/init"                  22 seconds ago   Up 9 seconds              514/tcp, 0.0.0.0:162->162/tcp, 0.0.0.0:162->162/udp, :::162->162/tcp, :::162->162/udp, 8000/tcp, 514/udp   librenms_snmptrapd
80a33e68c5ea   librenms/librenms:latest           "/init"                  22 seconds ago   Up 9 seconds              162/tcp, 8000/tcp, 162/udp, 0.0.0.0:514->514/tcp, 0.0.0.0:514->514/udp, :::514->514/tcp, :::514->514/udp   librenms_syslogng
8460a7fa0eb9   librenms/librenms:latest           "/init"                  22 seconds ago   Up 9 seconds              162/tcp, 162/udp, 514/tcp, 8000/tcp, 514/udp                                                               librenms_dispatcher
17b67d8600cc   librenms/librenms:latest           "/init"                  32 seconds ago   Up 10 seconds             162/tcp, 162/udp, 514/tcp, 514/udp, 0.0.0.0:8000->8000/tcp, :::8000->8000/tcp                              librenms
b2abf1fe4847   crazymax/msmtpd:latest             "/init"                  49 minutes ago   Up 31 minutes (healthy)   2500/tcp                                                                                                   librenms_msmtpd
4dede80fcaba   redis:5.0-alpine                   "docker-entrypoint.s…"   49 minutes ago   Up 31 minutes             6379/tcp                                                                                                   librenms_redis
65860dfee1fe   mariadb:10.5                       "docker-entrypoint.s…"   49 minutes ago   Up 31 minutes             3306/tcp                                                                                                   librenms_db
42e5ee62e633   quay.io/poseidon/matchbox:latest   "/matchbox -address=…"   11 days ago      Up 2 days                 0.0.0.0:8080-8081->8080-8081/tcp, :::8080-8081->8080-8081/tcp                                              laughing_germain
root@ip-172-26-1-22:/var/librenms# 
root@ip-172-26-1-22:/var/librenms# 
root@ip-172-26-1-22:/var/librenms# echo "<14>1 `date +'%Y-%m-%dT%H:%M:%S'` `hostname` TEST - - - This is a test syslog message" | nc -u -w1 127.0.0.1 514
root@ip-172-26-1-22:/var/librenms# docker logs 80a33e68c5ea
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-fix-logs.sh: executing... 
[cont-init.d] 00-fix-logs.sh: exited 0.
[cont-init.d] 01-fix-uidgid.sh: executing... 
[cont-init.d] 01-fix-uidgid.sh: exited 0.
[cont-init.d] 02-fix-perms.sh: executing... 
Fixing perms...
[cont-init.d] 02-fix-perms.sh: exited 0.
[cont-init.d] 03-config.sh: executing... 
Setting timezone to UTC...
Setting PHP-FPM configuration...
Setting PHP INI configuration...
Setting OpCache configuration...
Setting Nginx configuration...
Updating SNMP community...
Initializing LibreNMS files / folders...
Setting LibreNMS configuration...
Checking LibreNMS plugins...
Fixing perms...
Checking additional Monitoring plugins...
Checking alert templates...
[cont-init.d] 03-config.sh: exited 0.
[cont-init.d] 04-svc-main.sh: executing... 
[cont-init.d] 04-svc-main.sh: exited 0.
[cont-init.d] 05-svc-dispatcher.sh: executing... 
[cont-init.d] 05-svc-dispatcher.sh: exited 0.
[cont-init.d] 06-svc-syslogng.sh: executing... 
>>
>> Sidecar syslog-ng container detected
>>
[cont-init.d] 06-svc-syslogng.sh: exited 0.
[cont-init.d] 07-svc-cron.sh: executing... 
[cont-init.d] 07-svc-cron.sh: exited 0.
[cont-init.d] 08-svc-snmptrapd.sh: executing... 
[cont-init.d] 08-svc-snmptrapd.sh: exited 0.
[cont-init.d] ~-socklog: executing... 
[cont-init.d] ~-socklog: exited 0.
[cont-init.d] done.
[services.d] starting services
[2023-09-17T20:05:04.174290] WARNING: With use-dns(no), dns-cache() will be forced to 'no' too!;
[2023-09-17T20:05:04.176089] WARNING: window sizing for tcp sources were changed in syslog-ng 3.3, the configuration value was divided by the value of max-connections(). The result was too small, clamping to value of min_iw_size_per_reader. Ensure you have a proper log_fifo_size setting to avoid message loss.; orig_log_iw_size='3', new_log_iw_size='100', min_iw_size_per_reader='100', min_log_fifo_size='30000'
[services.d] done.
root@ip-172-26-1-22:/var/librenms# 
root@ip-172-26-1-22:/var/librenms# 
root@ip-172-26-1-22:/var/librenms# 
root@ip-172-26-1-22:/var/librenms# 
root@ip-172-26-1-22:/var/librenms# 
root@ip-172-26-1-22:/var/librenms# docker logs 17b67d8600cc
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-fix-logs.sh: executing... 
[cont-init.d] 00-fix-logs.sh: exited 0.
[cont-init.d] 01-fix-uidgid.sh: executing... 
[cont-init.d] 01-fix-uidgid.sh: exited 0.
[cont-init.d] 02-fix-perms.sh: executing... 
Fixing perms...
[cont-init.d] 02-fix-perms.sh: exited 0.
[cont-init.d] 03-config.sh: executing... 
Setting timezone to UTC...
Setting PHP-FPM configuration...
Setting PHP INI configuration...
Setting OpCache configuration...
Setting Nginx configuration...
Updating SNMP community...
Initializing LibreNMS files / folders...
Setting LibreNMS configuration...
Checking LibreNMS plugins...
Fixing perms...
Checking additional Monitoring plugins...
Checking alert templates...
[cont-init.d] 03-config.sh: exited 0.
[cont-init.d] 04-svc-main.sh: executing... 
Waiting 60s for database to be ready...
Database ready!
Updating database schema...

   INFO  Nothing to migrate.  


   INFO  Seeding database.  

  Database\Seeders\DefaultAlertTemplateSeeder ........................ RUNNING  
  Database\Seeders\DefaultAlertTemplateSeeder ................... 2.13 ms DONE  

  Database\Seeders\ConfigSeeder ...................................... RUNNING  
  Database\Seeders\ConfigSeeder ................................. 1.71 ms DONE  

Clear cache

   INFO  Application cache cleared successfully.  


   INFO  Configuration cached successfully.  

[cont-init.d] 04-svc-main.sh: exited 0.
[cont-init.d] 05-svc-dispatcher.sh: executing... 
[cont-init.d] 05-svc-dispatcher.sh: exited 0.
[cont-init.d] 06-svc-syslogng.sh: executing... 
[cont-init.d] 06-svc-syslogng.sh: exited 0.
[cont-init.d] 07-svc-cron.sh: executing... 
Creating LibreNMS daily.sh cron task with the following period fields: 15 0 * * *
Creating LibreNMS cron artisan schedule:run
Fixing crontabs permissions...
[cont-init.d] 07-svc-cron.sh: exited 0.
[cont-init.d] 08-svc-snmptrapd.sh: executing... 
[cont-init.d] 08-svc-snmptrapd.sh: exited 0.
[cont-init.d] ~-socklog: executing... 
[cont-init.d] ~-socklog: exited 0.
[cont-init.d] done.
[services.d] starting services
crond: crond (busybox 1.35.0) started, log level 8
2023/09/17 20:05:07 [notice] 620#620: using the "epoll" event method
2023/09/17 20:05:07 [notice] 620#620: nginx/1.22.1
2023/09/17 20:05:07 [notice] 620#620: OS: Linux 6.2.0-1011-aws
2023/09/17 20:05:07 [notice] 620#620: getrlimit(RLIMIT_NOFILE): 1048576:1048576
2023/09/17 20:05:07 [notice] 620#620: start worker processes
2023/09/17 20:05:07 [notice] 620#620: start worker process 635
2023/09/17 20:05:07 [notice] 620#620: start worker process 636
[services.d] done.
[17-Sep-2023 20:05:07] NOTICE: fpm is running, pid 617
[17-Sep-2023 20:05:07] NOTICE: ready to handle connections
52.41.73.66 - - [17/Sep/2023:20:05:22 +0000] "GET /validate HTTP/1.1" 200 9226 "http://52.25.143.176:8000/syslog" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.81"
52.41.73.66 - - [17/Sep/2023:20:05:23 +0000] "POST /ajax/set_resolution HTTP/1.1" 200 38 "http://52.25.143.176:8000/validate" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.81"
52.41.73.66 - - [17/Sep/2023:20:05:25 +0000] "GET /validate/results HTTP/1.1" 200 4046 "http://52.25.143.176:8000/validate" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.81"
2023/09/17 20:05:25 [info] 636#636: *1 client 52.41.73.66 closed keepalive connection
2023/09/17 20:05:25 [info] 636#636: *2 client 52.41.73.66 closed keepalive connection
52.41.73.66 - - [17/Sep/2023:20:05:42 +0000] "GET /validate HTTP/1.1" 200 9226 "http://52.25.143.176:8000/syslog" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.81"
52.41.73.66 - - [17/Sep/2023:20:05:42 +0000] "POST /ajax/set_resolution HTTP/1.1" 200 38 "http://52.25.143.176:8000/validate" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.81"
52.41.73.66 - - [17/Sep/2023:20:05:43 +0000] "GET /validate/results HTTP/1.1" 200 4046 "http://52.25.143.176:8000/validate" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.81"
2023/09/17 20:05:45 [info] 636#636: *7 client closed connection while waiting for request, client: 52.41.73.66, server: 0.0.0.0:8000
2023/09/17 20:05:45 [info] 636#636: *6 client 52.41.73.66 closed keepalive connection
2023/09/17 20:05:55 [info] 636#636: *11 client closed connection while waiting for request, client: 52.41.73.66, server: 0.0.0.0:8000
52.41.73.66 - - [17/Sep/2023:20:05:58 +0000] "GET /syslog HTTP/1.1" 200 9098 "http://52.25.143.176:8000/validate" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.81"
52.41.73.66 - - [17/Sep/2023:20:05:58 +0000] "POST /ajax/set_resolution HTTP/1.1" 200 38 "http://52.25.143.176:8000/syslog" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.81"
52.41.73.66 - - [17/Sep/2023:20:05:58 +0000] "POST /ajax/table/syslog HTTP/1.1" 200 57 "http://52.25.143.176:8000/syslog" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.81"
crond: USER librenms pid 779 cmd php /opt/librenms/artisan schedule:run --no-ansi --no-interaction > /dev/null 2>&1
2023/09/17 20:06:00 [info] 636#636: *13 client closed connection while waiting for request, client: 52.41.73.66, server: 0.0.0.0:8000
2023/09/17 20:06:00 [info] 636#636: *12 client 52.41.73.66 closed keepalive connection
crond: USER librenms pid 782 cmd php /opt/librenms/artisan schedule:run --no-ansi --no-interaction > /dev/null 2>&1
```

### Additional info

validation results from librenms gui

ConfigurationOk
DatabaseOk
DependenciesOk
DiskOk
DistributedpollerOk
MailOk
PhpOk
PollerOk
ProgramsWarning
WARN: IPv6 is disabled on your server, you will not be able to add IPv6 devices.
PythonOk
RrdOk
RrdcheckOk
SchedulerOk
SystemOk
UpdatesWarning
WARN: Non-git install, updates are manual or from package
UserOk
WebserverFailure
FAIL: ServerName is set incorrectly for your webserver, update your webserver config. 52.25.143.176 parse failure (/validate/results)
Fix:
ServerName 52.25.143.176

**Comment by melbruki** [comment]
ya eventually they showed up, give it some time. did you add the syslog server to librenms?

**Comment by huntson** [comment]
Did not eventually show up here.  I presume you mean this by saying 'add the syslog sever:

      - "SIDECAR_SYSLOGNG=1"

This has been set since I started the installation.

**Comment by huntson** [comment]
Any other thoughts here?

**Comment by Olen** [comment]
There is a problem with the syslog-container, but I have not been able to figure out the root cause.

When the container starts, two different `syslog-ng -F` processes are started.

One of the processes (the last to be started) is using 100% CPU, and spews "close"-syscalls if you strace the pid:

```
    PID USER      PR  NI    VIRT    RES    SHR S  %CPU  %MEM     TIME+ COMMAND                                                                                                                
   1941 root      20   0   18492    696    312 R 100.0   0.0   2:55.13 syslog-ng                                                                                                              
```

```
librenms_syslogng:/opt/librenms# strace -p 1941                                                                                                                                               
strace: Process 1941 attached                                                                  
close(140293650)                        = -1 EBADF (Bad file descriptor)   
close(140293649)                        = -1 EBADF (Bad file descriptor)                       
close(140293648)                        = -1 EBADF (Bad file descriptor)                                                                                                                      
close(140293647)                        = -1 EBADF (Bad file descriptor)                                                                                                                      
close(140293646)                        = -1 EBADF (Bad file descriptor)                       
close(140293645)                        = -1 EBADF (Bad file descriptor)                       
close(140293644)                        = -1 EBADF (Bad file descriptor)                       
close(140293643)                        = -1 EBADF (Bad file descriptor)               
```

As you can see, the FDs are counting down.  When that one reaches 0, it will either spawn a new syslog-ng process and start over again counting down from a gazillion or somethinh, or it will stop, and the container will start accepting syslog-messages.

So basically, it can take from a few minutes, to a few hours before the container starts accepting and parsing log-messages.

I have not found any similar bug reports anywhere, but it sounds like the same thing you experience.
